src/ros_filter.cpp

```diff
@@ -361,6 +361,25 @@ namespace RobotLocalization
   }
 
   template<typename T>
+  std::string RosFilter<T>::tf2NameSanitizer(const std::string & param )
+  {
+    if(!tfPrefix_.empty())
+    { // tf prefix is set
+      if(param.at(0)=='/')
+        return tfPrefix_ + param;
```

Comment:
Please use curly braces for all conditional statements, and use line breaks:

if(foo)
{
  statement;
}